vital/bindings/python/vital/types/descriptor_set.cxx

```diff
@@ -30,17 +30,41 @@
 
 #include <pybind11/stl.h>
 
-#include "descriptor_class.cxx"
+#include <vital/types/descriptor_set.h>
 
 namespace py = pybind11;
 
+typedef kwiver::vital::descriptor_set desc_set;
+typedef kwiver::vital::simple_descriptor_set s_desc_set;
+
+std::shared_ptr<s_desc_set>
+new_desc_set()
+{
+  return std::shared_ptr<s_desc_set>(new s_desc_set());
```

Comment:
Would std::make_shared<>() be better here?

Reply:
no significant preference, I'll make the change since it shortens code